usr.bin/renice/renice.c

```diff
@@ -97,12 +97,13 @@ main(int argc, char *argv[])
 			if ((pwd = getpwnam(*argv)) != NULL)
 				who = pwd->pw_uid;
 			else if (getnum("uid", *argv, &who)) {
-				errs++;
-				continue;
+				warnx("invalid user name or uid: %s", *argv);
+				errno = EINVAL;
+				err("invalid user name or uid: %s", *argv);
```

Comment:
So this prints the error twice.
And assumes that the error text for EINVAL is the right thing to give the user.
The old warning was fine.
Also, err takes an integer first parameter that's the exit value, so this code couldn't have even compiled.